C/common/storage_client.cpp

```diff
@@ -1365,3 +1365,26 @@ void StorageClient::handleException(const exception& ex, const char *operation,
 		}
 	}
 }
+
+/**
+ * Post to storage schema
+ */
+bool StorageClient::postStorageSchema(Reading& reading)
```

Comment:
What is this here for? I don't understand why you are sending a reading? The URL should be creating a schema which certainly doesn't take a reading as an argument or as the payload to the POST. Also, please name this for the meaning to the user and not the mechanics of what happens. If it is creating a schema call it createSchema, the fact is is doing a HTTP POST request is irrelevant to the user and if we ever change to say MQTT as a transport is also then incorrect.

Reply:
Fixed.